modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -7271,6 +7271,8 @@ feature.flag.COMMERCE-13024.description=Users can view and edit custom object fi
 feature.flag.COMMERCE-13024.title=Organization Management Widget and Custom Object Fields
 feature.flag.LPD-6368.description=Users can scope search result rankings (i.e., everything, site, or blueprint). Rankings only apply to searches with a matching scope. When enabled, existing rankings apply to searches with the everything scope. When disabled, result rankings with the site or blueprint scope disappear.
 feature.flag.LPD-6368.title=Scoping Result Rankings
+feature.flag.LPD-10793.description=Generate and save AI images in the Documents & Media application using OpenAI.
+feature.flag.LPD-10793.title=AI image generation in DAM
```

Comment:
Titles should be capitalized: _AI Image Generation in DAM_

Reply:
Updated, thank you!